python/lsst/validate/base/__init__.py

```diff
@@ -5,15 +5,15 @@
 
 try:
     # version.py is auto-generated by sconsUtils
-    from .version import *  # noqa: F403
+    from .version import *  # noqa: F401, F403
```

Comment:
I would think that these are so obvious and non-controversial that you can just do a PR, and I'll sign off.